ouroboros-consensus/src/Ouroboros/Consensus/Block/Forging.hs

```diff
@@ -218,20 +211,8 @@ checkShouldForge BlockForging{..}
 -------------------------------------------------------------------------------}
 
 -- | The result of updating something, e.g., the forge state.
-data UpdateInfo updated unchanged failed =
-    Updated      updated
-  | Unchanged    unchanged
+data UpdateInfo updated failed =
+    Updated updated
+    -- ^ The update have induced no change.
```

Comment:
*has

Reply:
I'm confused as to how you even saw that line: it's not present on `4b22efadc`, which is -- and I think always has been -- the tip of this PR.